src/sources/kafka.rs

```diff
@@ -677,6 +619,7 @@ mod integration_test {
     }
 
     #[tokio::test]
+    #[ignore]
```

Comment:
Do we still want this test, just with a different behavior?

Reply:
I _think_ so, which is why I didn't outright remove the test. I was having difficulty getting the test to run before I submitted it due to container issues. I'll take another look.